_announcements/week-04.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Week 4 Review 
+week: 4
+date: 2023-02-3
+---
+
+# 🍀  Week 4: Making decisions in programs (Branching)
+
+Dear all,
+
+We’re almost halfway through the quarter!
+
+- During class and lab, we discussed different data structures such as:
+    - Conditionals
+    - Booleans and Boolean Operators: Comparison Operators and Logical Operators
+    - Branching using the `if`-`elif`-`else` statements
+    - `assert` statements for writing tests of return values
+        
+* The slides and class recordings as well as the videos from the class and lab sections are posted on Gauchospace (look for the ⏩ icon; use your <span style="color:red">**UCSB address**</span> - we won't be granting access to personal gmail accounts.)
+
+* The last section of every chapter in zyBook contains our class notes, which we update afterward with whatever was discussed in class. Sometimes, like this week, there is additional code for you to practice with (the number-guessing game using the `random` module).
+
+## ⏰ Week 5 - Due dates and Deadlines ⏰
+
+Check the [Calendar]({{site.url}}/{{site.baseurl}}/calendar#week-4), for the due dates. They are the same as they've been before.
+
+Upcoming deadlines are:
+* ⏰  **Tuesday, 2/07 11:59PM**: **LA04**
+* ⏰  **Wednesday, 2/08 11:59PM**: **LA05 CHECKPOINT** Week 5 zyLabs with the word “Checkpoint” in their title
+* ⏰ **Thursday, 2/09**: in-class Quiz 4 on Gauchospace, covering Week 1-4 content.
```

Comment:
No in-class quiz this week to ensure we have enough time to review using IDLE, answer questions and practice using Gradescope.